translator/translate/otel/pipeline/prometheus/translator.go

```diff
@@ -58,10 +61,36 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 	}
 
 	// return pipeline based on destination to keep source/destination combinations clearly separated
+	// otel_prometheus - cloudwatch
+	// telegraf_prometheus - cloudwatch logs
 	// telegraf_prometheus - cloudwatch
```

Comment:
nit: Makes it seem like the telegraf prometheus plugin can send to CW/PMD and CWL/PLE when that is not the case.